doc/nrf/getting_started.rst

```diff
@@ -6,7 +6,9 @@ Getting started
 To quickly get started with the |NCS|, use :ref:`nRF Connect for Desktop <gs_assistant>` to set up your development environment.
 Alternatively, check the :ref:`gs_installing` section for instructions on setting up your environment manually.
 
-We recommend using SEGGER Embedded Studio for building your applications. See :ref:`gs_programming` for the download links and instructions.
+We recommend using the `nRF Connect for Visual Studio Code`_ extension or |SES| to build applications.
```

Comment:
Me, @melwee01 , @Avalei , @b-gent , and @umapraseeda  had a brief call about this PR and the VS Code release in the NCS context. Things discussed:

- We would like this PR to go into the NCS 1.7.0 release.
- We think updates to samples about support for VSC can be made after the NCS release.

About the second point, we wonder if it would make sense to add a note stating that the samples might not be entirely compatible with VSC just yet (to avoid issues such as imcompatibility with the Thread samples after the 1.7.0 is released). However, this should be decided by you, @wbober , @ihansse , and the PMT.

Reply:
@greg-fer We added a note on line 13 to clarify the "incompatibility". The samples will actually work, but may require the customer to alter the directory tree and configuration for it to work with the VSC extension. We do not currently have detailed troubleshooting instructions for all samples since the individual teams are still testing their samples. So this note will be meant as a catch-all warning.

Reply:
```suggestion
We recommend using the |VSC| or |SES| to build applications.
```